CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # beeline-ruby changelog
 
+## 3.1.0 2024-03-07
+
+### Enhancements
+
+- feat: support classic ingest keys (#238) | Robb Kidd
```

Comment:
Unclear if we want to put a note in about the min. libhoney version bump?

Reply:
Probably a good idea. I don't _expect_ that to hurt anyone. 🤞🏻 😅 🤞🏻  

Maybe as a note for the release above the **Enhancements** section?

Reply:
Done 👍🏻